doc/nrf/releases/release-notes-0.4.0.rst

```diff
@@ -131,7 +131,7 @@ Bluetooth Low Energy
 
 * Added the following libraries:
 
-  * :ref:`dfu_smp_c_readme` - GATT Client implementation compatible with the GATT SMP Service, which is used as DFU transport
+  * :ref:`dfu_smp_readme` - GATT Client implementation compatible with the GATT SMP Service, which is used as DFU transport
```

Comment:
Nothing wrong on the line, but I cannot place a comment on the commit message:
Commit: https://github.com/nrfconnect/sdk-nrf/pull/3066/commits/744b836cf5928875f0b047095c67614ebe52b993 states:
```
 Bluetooth: services: DFU SMP: Align DFU SMP service name convention

Align the BAS service naming with upstream service convention.
```
I think `BAS` should have been `DFU AMP` in the commit message.

Reply:
Thank you. Fixed.